packages/dashboard/src/Dashboard.vue

```diff
@@ -252,9 +252,24 @@ export default class Dashboard extends Vue {
   accounts: accountInterface[] = [];
   loadingAPI = true;
   version = config.version;
-
+  async subscribe() {
+    await this.$store.dispatch("portal/subscribeAccounts").then(async extensions => {
```

Comment:
Can we use the mutation types instead of hard-coded ones?
e.g.

```suggestion

    // in the mutations file
    export enum MutationTypes {
        SUBSCRIBE_ACCOUNTS = 'subscribeAccounts'
    }
    await this.$store.dispatch(`portal/${ActionTypes.SUBSCRIBE_ACCOUNTS}`);
```

also the `portal` string can be added in the enum itself if it is not used somewhere else.